inference-engine/tests/functional/plugin/cpu/shared_tests_instances/skip_tests_config.cpp

```diff
@@ -56,6 +56,6 @@ std::vector<std::string> disabledTestPatterns() {
         // TODO: not supported yet, ticket 37690
         R"(.*Loop.*)",
         // TODO: Issue: 41694
-        R"(.*smoke_Set2.*CTCLossLayerTest.*)",
+        R"(.*CTCLossLayerTest.*CPU*)",
```

Comment:
I offer to use `R"(.*CTCLossLayerTest.*)",`. More over, are you really want to disable all `R"(.*CTCLossLayerTest.*CPU*)",`?

Reply:
Yes, there are also failures on smoke_Set1: https://dev.azure.com/openvinoci/dldt/_build/results?buildId=19754&view=logs&j=a82d9e72-1de4-5bd6-c188-90296135479e&t=fcb08720-f924-573e-e2ce-d8d2f68bf743